data_processing/data_processing.py

```diff
@@ -23,3 +29,186 @@ def data_processing(video_path: Path, output_path: Path) -> List[Path]:
 
```

Comment:
Separate images should be called after `crop_and_resize_images()` here. The function should also take in `binary` as a boolean parameter which could be an additional parameter from the command line.

Reply:
I added the code to do so. However, when I run `data_processing()` I get an error from the `crop_and_resize_images()` part. I have added the code, but can't test the behaviour at the moment.